### PR TITLE
Runcards qibolab paper

### PR DESCRIPTION
**Pull request by Jacfomg**
I think those are the scans we did. 

Checklist:
- [ ] Reviewers confirm new code works as expected.
- [ ] Tests are passing.
- [ ] Coverage does not decrease.
- [ ] Documentation is updated.
- [ ] Compatibility with Qibo modules (Please edit this section if the current pull request is not compatible with the following branches).
    - [ ] Qibo: `master`
    - [ ] Qibolab: `main`
    - [ ] Qibolab_platforms_qrc: `main`

### CODE REVIEW

File: runcards/scaling/1d_sweepers.yml [hunk 1]
```diff
@@ -0,0 +1,292 @@
+platform: iqm5q
```
Comment (stavros11) [inline runcards/scaling/1d_sweepers.yml:1]: Here maybe we should put
```suggestion
platform: dummy
```
to make it neutral, like the other file.

Also, I am not sure if we really need the folder as it only contains one file. Maybe we can just rename it `scaling_benchmarks.yml` or something like that and leave it under runcards.

**Review by stavros11 — COMMENTED** [review]
Thanks for opening this. The scaling runcard is the same with the one I used, but `benchmarks.yml` is different. I believe in the end we used [benchmark_iqm5q_q0.yml](https://github.com/qiboteam/qibocal/pull/433/files#diff-f5b55b1130a20d311f2933901ccacce2b68103dcd30558677835be918553ba20) from #433, not [benchmark.yml](https://github.com/qiboteam/qibocal/pull/433/files#diff-c3e6f1df1daecd9fdbc965ba645a3669da52f82c7311408c6ab242a934ea7de4) that is copied here.

File: runcards/scaling/1d_sweepers.yml (hunk 1 above)
Comment (Jacfomg) [reply runcards/scaling/1d_sweepers.yml:1]: I left just in case we go for a 2d_sweepers thing in the future, but yeah right now it's a bit pointless.

Comment (stavros11) [reply runcards/scaling/1d_sweepers.yml:1]: Sure, that is fine with me. Thanks for the updates.

**Review by stavros11 — APPROVED** [review]

File: runcards/scaling/1d_sweepers.yml (hunk 1 above)
Comment (Jacfomg) [reply runcards/scaling/1d_sweepers.yml:1]: I will add the dummy as well when I can push

**Review by andrea-pasquale — APPROVED** [review]
Thanks @Jacfomg @stavros11

**Pull request merged**